lucien-ios/Dashboard/DashboardComicCells/CollectionViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LendingCollectionViewCell.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/10/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class CollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet private weak var dueDateLabel: UILabel!
+    @IBOutlet weak var ownerBorrowerNameLabel: UILabel!
+    @IBOutlet weak var bookImage: UIImageView!
+    @IBOutlet weak var lentOrBorrowedLabel: UILabel!
+    @IBOutlet weak var ownerOrBorrowerLabel: UILabel!
+
+    static let reuseIdentifier = "comicCell"
+
+    func configure(userType: String, comicDueDate: Date?, ownerBorrowerName: String, imageURL: String?) {
+        dueDateLabel.text = formatDateToString(date: comicDueDate)
+        configureLentOrBorrowedLabel(userType: userType)
+        ownerBorrowerNameLabel.text = ownerBorrowerName
+        setImage(imageURL: imageURL)
+        setStyling()
+    }
+
+    func configureLentOrBorrowedLabel(userType: String) {
+        switch userType {
+        case "borrower":
+            lentOrBorrowedLabel.text = "LENT TO"
```

Comment:
This text should come from the configure method, instead of passing in the user type and determining it from there